example-document/03-troubleshooting.md

```diff
@@ -0,0 +1,83 @@
+# Troubleshooting {#troubleshooting}
+
+Many pipeline failures are caused by justa few issues. This section describes the debugging process of a failed pipeline.
+
+First of all, to see the pipeline statu:
+1. Go to your repository
+2. Open Actions tab
+3. Open detail of your pipeline run
+4. Find out what job is failing
+5. See detailed logs of failed job
+6. Seach for commonly known error messages using native GitHub log search on top right corner
+
+## Validate YAML documents
+
+YAML formatting is strict, and all errors in this job are related to it. Search for `yaml.scanner.ScannerError` to see what went wrong. Example of the log:
+
+```
+yaml.scanner.ScannerError: mapping values are not allowed in this context
+  in "/__w/istqb-ctal-ta/istqb-ctal-ta/sample-exam-answers/questions.yml", line 562, column 19
+```
```

Comment:
This code example overflows:

![image](https://github.com/user-attachments/assets/f39951db-2829-4da1-b3e4-76b0ace537cd)

In general, code examples should not exceed 90 characters per line.

Here, I suggest shrinking the path:

````suggestion
```
yaml.scanner.ScannerError: mapping values are not allowed in this context
  in "istqb-ctal-ta/sample-exam-answers/questions.yml", line 562, column 19
```
````

Reply:
Fixed